notes/stories/film/film-scifi-1990s.st.txt

```diff
@@ -11669,6 +11669,47 @@ factions joining forces against a common enemy [The people of Naboo joined force
 telekinetic ability [Obi-Wan used the power of his own mind to move a light saber to his hand in order to defeat Darth Maul.]
 
 
+movie: Deep Blue Sea (1999)
+===========================
+
+:: Title
+Deep Blue Sea
+
+:: Description
+Set in an isolated underwater facility, the film follows a team of scientists
+and their research on mako sharks to help fight Alzheimer's disease. The
+situation plunges into chaos when multiple genetically engineered sharks go
+on a rampage and flood the facility.
+
+:: Date
+1999-07-28
+
+:: References
+https://en.wikipedia.org/wiki/Deep_Blue_Sea_(1999_film)
+
+:: Ratings
+2 <paul>
+
+:: Genre
+sci-fi
+action
+
+:: Choice Themes
+human vs. beast [The people trapped in the ocean facility were being terrorized by multiple genetically engineered sharks.]
+
+:: Major Themes
+cure for an as yet uncurable disease [Dr. Susan McCallister was genetically engineering mako sharks to have bigger brains as part of her research to help cure Alzheimer's disease.]
+the dangers of unfettered scientific advancement [Dr. Susan McCallister accidentally created a pack of intelligent sharks that swam amok while seeking a cure for Alzheimer's disease.]
```

Comment:
Were the sharks searching for a cure for Alzheimer's?

Reply:
I wish. That would have put an interesting spin on things. But alas not. Comment disambiguated accordingly.